File: src/main/java/com/thebluehats/server/game/utils/RomanNumeralConverter.java
package com.thebluehats.server.game.utils;

public class RomanNumeralConverter {
    public String convertToRomanNumeral(int value) {
        switch (value) {
            case 0:
                return "None";
            case 1:
                return "I";
            case 2:
                return "II";
            case 3:
                return "III";
            case 4:
                return "IV";
            case 5:
                return "V";
            case 6:
                return "VI";
            case 7:
                return "VII";
            case 8:
                return "VIII";
            case 9:
                return "IX";
            case 10:
                return "X";
            case 11:
                return "XI";
            case 12:
                return "XII";
            case 13:
                return "XIII";
            case 14:
                return "XIV";
            case 15:
                return "XV";
            case 16:
                return "XVI";
            case 17:
                return "XVII";
            case 18:
                return "XVIII";
            case 19:
                return "XIX";
            case 20:
                return "XX";
            case 21:
                return "XXI";
            case 22:
                return "XXII";
            case 23:
                return "XXIII";
            case 24:
                return "XXIV";
            case 25:
                return "XXV";
            case 26:
                return "XXVI";
            case 27:
                return "XXVII";
            case 28:
                return "XXVIII";
            case 29:
                return "XXIX";
            case 30:
                return "XXX";
            case 35:
                return "XXXV";
        }

        return null;
    }

    public int convertRomanNumeralToInteger(String numeral) {
        switch (numeral) {
            case "I":
                return 1;
            case "II":
                return 2;
            case "III":
                return 3;
        }

        return -1;
    }
}

File: src/main/java/com/thebluehats/server/game/utils/RomanNumeralConverter.kt
package com.thebluehats.server.game.utils

class RomanNumeralConverter {
    fun convertToRomanNumeral(value: Int): String? {
        when (value) {
            0 -> return "None"
            1 -> return "I"
            2 -> return "II"
            3 -> return "III"
            4 -> return "IV"
            5 -> return "V"
            6 -> return "VI"
            7 -> return "VII"
            8 -> return "VIII"
            9 -> return "IX"
            10 -> return "X"
            11 -> return "XI"
            12 -> return "XII"
            13 -> return "XIII"
            14 -> return "XIV"
            15 -> return "XV"
            16 -> return "XVI"
            17 -> return "XVII"
            18 -> return "XVIII"
            19 -> return "XIX"
            20 -> return "XX"
            21 -> return "XXI"
            22 -> return "XXII"
            23 -> return "XXIII"
            24 -> return "XXIV"
            25 -> return "XXV"
            26 -> return "XXVI"
            27 -> return "XXVII"
            28 -> return "XXVIII"
            29 -> return "XXIX"
            30 -> return "XXX"
            35 -> return "XXXV"
        }

        return null
    }

    fun convertRomanNumeralToInteger(numeral: String?): Int {
        when (numeral) {
            "I" -> return 1
            "II" -> return 2
            "III" -> return 3
        }

        return -1
    }
}